### PR TITLE
Upload tck user documentation concurrency 3.1

### PR DESCRIPTION
**Pull request by KyleAure**
Add instructions to the `io.openliberty.jakarta.concurrency.3.1_fat_tck` to allow external users to verify the concurrency reference implementation.

### CODE REVIEW

**Review by njr-11 — DISMISSED** [review]

File: dev/io.openliberty.jakarta.concurrency.3.1_fat_tck/build.gradle [hunk 1]
```diff
@@ -11,4 +11,91 @@
...
+  arquillian
+  wlp
+}
+
+dependencies {
+  arquillian 'io.openliberty.arquillian:arquillian-liberty-support-jakarta:2.1.1'
+
+  //TODO Configure WLP dependency if doing verification testing, Examples below:
+  //wlp 'openliberty:runtime:openliberty-24.0.0.6-beta-cl240520240423-2000@zip' // certification version
+  //wlp 'io.openliberty.beta:openliberty-runtime:24.0.0.5-beta@zip'             // future versions
```
Comment (njr-11) [inline dev/io.openliberty.jakarta.concurrency.3.1_fat_tck/build.gradle]:
```suggestion
  //wlp 'io.openliberty.beta:openliberty-runtime:24.0.0.6-beta@zip'             // future versions
```

**Review by njr-11 — DISMISSED** [review]

**Comment by KyleAure** [comment]
#libby - check headers 
#build - personal build

**Comment by KyleAure** [comment]
#libby - recheck headers

**Review by njr-11 — DISMISSED** [review]

**Comment by KyleAure** [comment]
#libby - re-check headers
#build - personal build

**Review by njr-11 — APPROVED** [review]

**Pull request merged**